bin/companion

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+sh ../env.sh
+
+export UPPYSERVER_DATADIR="./packages/@uppy/companion/output"
+export UPPYSERVER_DOMAIN="localhost:3020"
+export UPPYSERVER_PROTOCOL="http"
+export UPPYSERVER_PORT=3020
+export UPPY_ENDPOINTS=""
+export UPPYSERVER_SECRET="development"
+
+nodemon --exec node ./packages/@uppy/companion/src/standalone/start-server.js
```

Comment:
nodemon watches everything in the working directory by default, so this will restart each time any Uppy client file is built or updated too. (try doing `npm start` in the repo root.) Using `--watch packages/@uppy/companion` would help there

Reply:
Instead of exporting, e.g. 

```
env \
  UPPYSERVER_SECRET="development" \
nodemon --exec node ./packages/@uppy/companion/src/standalone/start-server.js
```

seems a lot cleaner.

But bundling env without the possibility to override inside a bin seems wrong to me. Instead, we should consider to make these dev settings the default config inside the js, and just have a `start:dev` script that says `nodemon companion`, and rely on the package.json bin reference to `standalone/start-server.js`. For production, we would then use env vars to override the bundled dev settings.

Reply:
> Instead, we should consider to remove this shell file, make these dev settings the default config inside the js, and have a start:dev script that says nodemon companion, which relies on the package.json bin referencing standalone/start-server.js.

@kvz the `start:companion` command itself is considered for dev use only. It runs the src file directly and not the built (`./lib`) version.

I do agree that the side-effects from exporting is a bad idea, so I have gone with `env` as you suggested. But please let me know if you have other reasons for us not to go with this.